src/com/company/Team.java

```diff
@@ -0,0 +1,70 @@
+package com.company;
+
+import java.util.ArrayList;
+
+public class Team {
+    private int teamId;
+    private String name;
+    private int noOfPlayers;
+    private int score=0;
+    private int notOut;
+    ArrayList<Player> member = new ArrayList<>();
+
+    public void setId(int id)
+    {
+        teamId = id;
+    }
+    public int getId() {
+        return teamId;
+    }
+    public void setPlayers(int n){
+        noOfPlayers = n;
+    }
+    public int getPlayers(){
+        return noOfPlayers;
+    }
+
+    public void setName(String nameTeam)
+    {
+        name = nameTeam;
+    }
+    public String getName(){
+        return name;
+    }
+
+    public void setScore(int runs){
+        score = runs;
+    }
+    public int getScore(){
+        return score;
+    }
+    public int getNotOut(){
+        return notOut;
+    }
+    public void setNotOut(int num){
+        notOut = num;
+    }
+
+    public void setupPlayers(int idCode){
+        for(int i=0;i<noOfPlayers;i++)
+        {
+            Player p = new Player();
+            p.name = name + (i+1);
```

Comment:
where are you initializing name with?

Reply:
To make things easy, I assigned all players names as team name plus an integer. For eg: CSK players are CSK1, CSK2 etc. So, this name variable is representing team name and then I am adding integer to it and storing it as players's name.